docs/publishing.md

```diff
@@ -1,37 +1,46 @@
-- You have cloned the mdn/browser-compat-data repository and you have the latest `main` branch checked out.
-- You have permissions to merge pull requests and publish releases.
-- You have [the `gh` CLI](https://cli.github.com/) installed.
-- You have [`jq`](https://stedolan.github.io/jq/) installed.
+1. The [`release-pr` workflow](https://github.com/mdn/browser-compat-data/blob/main/.github/workflows/release-pr.yml) manages a pull request draft that bumps the `package.json` version and updates the release notes.
+2. The [`release` workflow](https://github.com/mdn/browser-compat-data/blob/main/.github/workflows/release.yml) creates the GitHub release and tag, and publishes the release on NPM.
 
 ## Performing a release
 
-Releasing a new version of BCD is a two-step process: first, the release script is run to bump version number, generate the changelog, and create a pull request; then, once the pull request is merged, a release on GitHub is performed.
+A release is usually performed by merging the pull request managed by the `release-pr` workflow:
 
-First, run `npm run release`. This will automatically perform all of the steps needed to release a new version of BCD. If user input is needed, the script will announce as such. Once the script is complete, a pull request to release a new version of BCD will be created from your GitHub account, which can be reviewed and merged.
+1. The merge commit will trigger the `create-release` job of the `release` workflow.
+2. The `create-release` job publishes a [GitHub release](https://github.com/mdn/browser-compat-data/releases) and [tag](https://github.com/mdn/browser-compat-data/tags) based on the updated `package.json` and release notes.
+3. The published GitHub release will trigger the `publish-release` job of the `release` workflow.
+4. The `publish-release` job performs the following steps:
+   - Builds the release.
+   - Publishes the release to NPM.
+   - Adds the `data.json` as an asset to the GitHub release.
+   - Dispatches a `bcd_release` event on the [mdn/bcd-utils](https://github.com/mdn/bcd-utils) repository, which triggers the deployment of the new BCD release on MDN.
 
-Once you have gotten the applicable review on your PR, it's time to finish the release process:
+### Before merging the release PR
 
-1. Merge the pull request created by the script.
+> [!WARNING]
+>
+> To ensure the release PR is up-to-date, care should be taken that no other PR is merged at the same time, and that all checks have passed on the latest `main` commit.
 
-2. Start a [release on GitHub](https://github.com/mdn/browser-compat-data/releases).
+> [!TIP]
+>
+> For major and minor releases, the `RELEASE_NOTES.md` can be edited manually on the `release` branch. However, as the next `release-pr` workflow run will overwrite the changes, it is advisable to keep a copy, or to disable the workflow temporarily.
```

Comment:
This note should go _before_ the merge step or, better still, integrated into the text as an conditional step (e.g., "1. If this is a major and minor release, then…. 2. Merge the pull request.")

Reply:
Done.